index.js

```diff
@@ -76,12 +87,14 @@ const getArrows = arrows => arrows && (
 );
 
 const getPagination = pagination => pagination && <div class='swiper-pagination'/>;
-const afterMount = ({props}, el) => requestAnimationFrame(() => init({props}, el));
-const shouldUpdate = ({props}, nextProps) => !deepEqual(props, nextProps);
+
+const afterRender = ({props}, el) => {
+	requestAnimationFrame(() => init({props}, el));
+};
 
 const render = ({props}) => {
 	const {arrows, children, pagination} = props;
-	const items = children.map(x => <div class='swiper-slide'>{x}</div>);
+	const items = children.map((x, i) => <div key={i} class='swiper-slide'>{x}</div>);
```

Comment:
I am afraid this might cause issues. Since `i` isn't unique for the actual slide, just the placement in the array.